mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -1825,4 +1825,42 @@ def LLVM_CallIntrinsicOp
   let hasVerifier = 1;
 }
 
+def LLVM_LinkerOptionsOp
```

Comment:
Can we add a verifier that checks that this on directly inside a module. Similar to the verifier of `GlobalOp`?